### ISSUE TITLE
git push tags in command palette

### ISSUE THREAD

**Issue by randy3k** [divmain/GitSavvy#326]
While "git: tags" gives an integrated way to manage tags, the only time I really need it is to push tags.
@divmain, how do you think commands like "git: push all tags" or "git: push selected tag" in the command palette?

**Comment by divmain** [comment]
`git: push all tags` should be extremely simple to add, but it adds clutter to the more commonly used `git: push ...` commands.  If it were just me, I would stick to the tags dashboard, since I like seeing what tags I'm pushing anyway.  However, others might prefer a different work flow.

Let me know your preference, and let's see if anyone else chimes in in support of this.  In the meantime, if you just want something you can use, you can create your own `.sublime-commands` file in the `User/` directory with the following:

```json
    {
        "caption": "git: push all tags",
        "command": "gs_tag_push",
        "args": { "push_all": true }
    }
```

**Comment by randy3k** [comment]
Thanks. `git: push all tags` works well. I also agree that "git: push ...." may not too much since we have the tags dashboard.

I have just submitted a PR on `git: push all tags`, it is something that I definitely need.

**Comment by randy3k** [comment]
#332 is tracking this now.

**Comment by divmain** [comment]
Thanks!

In the future, though, please write something like `Closes #326.` in the description of the PR.  GitHub will then auto-close the issue when the PR is merged (you can also do this in the commit message).  This ensures that, if the PR isn't accepted, we don't lose track of the original issue.  Plus it ties the issue directly to the remedy :)

**Comment by randy3k** [comment]
Thanks for the fast response.
I should have probably done that. :)
